src/pages/markets/_hero.js

```diff
@@ -8,6 +8,19 @@ import { Container } from 'components/containers'
 import { useLazyVideo } from 'components/hooks/lazy-video'
 import device from 'themes/device'
 
+const MarketHeader = styled(Header)`
+    font-size: 64px;
+    @media ${device.tabletL} {
+        font-size: 40px !important;
```

Comment:
Is it a must to add the important tag here?

Reply:
fixed